benchmark/cublas/main.cpp

```diff
@@ -0,0 +1,35 @@
+#include "cublas_v2.h"
```

Comment:
Could you please add the license at the beginning of this file, the `CMakeLists.txt` file and the `utils.hpp` file?